daq_macros.py

```diff
@@ -3258,7 +3290,37 @@ def loop_center_xrec():
   #now try to get the loopshape starting from here
   return 1
 
-  
+def xrec_no_zebra(angle_start):
+  print(f'xrec_no_zebra{angle_start}')
+  beamline_lib.mvaDescriptor("omega", angle_start)
+  #yield from bps.mv(samplexyz.omega, angle_start)
+  for omega_target in range (angle_start, angle_start+360, 40):
+    #yield from bps.mv(samplexyz.omega, omega_target)
+    beamline_lib.mvaDescriptor("omega", omega_target)
+    logger.info(f'taking image at {omega_target}')
+    timeout = 5
+    # change image mode to single(0)
+    beamline_support.setPvValFromDescriptor("lowMagImMode",0)
```

Comment:
is there a reason why you this function name instead of setPvDesc? that alias is defined in the imports and makes it significantly easier to read

Reply:
I have fixed this with [d4fadc2](https://github.com/NSLS-II/lsdc/pull/314/commits/d4fadc2c96a80c281c5981a674a6373d791f4b6b)